Update/omot_008.txt

```diff
@@ -1460,7 +1460,7 @@ void main()
 //rいつもはにこやかな梨花ちゃんが足元の砂利を弄びながら、ここに来る途中からずっと硬く緊張した表情で俯いているのを見て、俺は緩んだ気持ちを引き締めていった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "いつもはにこやかな梨花ちゃんが足元の砂利を弄びながら、ここに来る途中からずっと硬く緊張した表情で俯いているのを見て、俺は緩んだ気持ちを引き締めていった。",
-			NULL, "Rika-chan was playing with the gravel by her feet, after wearing a tense expression the whole way here. When I saw that, my mood quickly tightened up.", Line_Normal);
+			NULL, "Rika-chan was playing with the gravel by her feet, after wearing a tense expression once we got here. When I saw that, my mood quickly darkened.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Rika-chan was playing with the gravel by her feet, wearing a tense expression since we got here. When I saw that, my mood quickly darkened.", Line_Normal);
```